web-admin/src/routes/[organization]/-/settings/billing/+page.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import {
+    createAdminServiceGetBillingSubscription,
+    createAdminServiceListOrganizationBillingIssues,
+  } from "@rilldata/web-admin/client";
+  import { mergedQueryStatus } from "@rilldata/web-admin/client/utils";
+  import Payment from "@rilldata/web-admin/features/billing/Payment.svelte";
+  import Plan from "@rilldata/web-admin/features/billing/plans/Plan.svelte";
+  import Spinner from "@rilldata/web-common/features/entity-management/Spinner.svelte";
+  import { EntityStatus } from "@rilldata/web-common/features/entity-management/types";
+  import type { PageData } from "./$types";
+
+  export let data: PageData;
+
+  $: ({ organization, showUpgrade } = data);
+
+  $: allStatus = mergedQueryStatus([
+    createAdminServiceGetBillingSubscription(organization),
+    createAdminServiceListOrganizationBillingIssues(organization),
+  ]);
+</script>
+
+{#if $allStatus.isLoading}
+  <Spinner status={EntityStatus.Running} size="16px" />
+{:else}
+  <Plan {organization} {showUpgrade} />
+  <Payment {organization} />
+{/if}
```

Comment:
Can we move the individual loading states into the `Plan` and `Payment` components? If we can't, mind adding a comment above `allStatus` to say why?

Reply:
It is better to show a single spinner than 2. Added a comment.